client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -117,9 +118,11 @@ const Header: FunctionComponent< Props > = ( {
 				</div>
 			</div>
 			<div className="gutenboarding__header-section">
-				<Link to={ next } className="gutenboarding__header-next-button" isPrimary isLarge>
-					{ NO__( 'Next' ) }
-				</Link>
+				{ next && hasSelectedDesign && (
```

Comment:
I don't like the idea of making the header more aware of the component and step logic and selecting more state. I'm not sure what the solution is right now 🤔

Reply:
Me neither, at this point. I think we'll reconsider as Gutenboarding continues to evolve. Also see my (lengthy :sweat_smile: ) comment here: https://github.com/Automattic/wp-calypso/pull/38705#discussion_r365201545